Logic.json

```diff
@@ -65,7 +65,7 @@
 		"Inferno": [["Clockworks", "Glitches"], ["Van"]],
 		"Saucer_Button": [["Clockworks",  "Glitches"], ["Witchyworld", "Egg_Aim", "Explosives"], ["Witchyworld", "Insane"]],
 		"Cliff_Top": [["Split_Up"]],
-		"Jolly_Rogers_Lagoon": [["Cliff_Top", "JiggyWiggySpecial"], ["Cliff_Top", "14_Jiggies"]],
+		"Jolly_Rogers_Lagoon": [["Cliff_Top", "JiggyWiggySpecial"], ["Cliff_Top", "14_Jiggies"], ["Hailfire_Peaks", "Split_Up", "Cloud_Cuckooland"], ["Hailfire_Peaks_B", "Cloud_Cuckooland"], ["Hailfire_Peaks", "Grip_Grab", "Glitches"]],

```

Comment:
["Hailfire_Peaks_B", "Cloud_Cuckooland"] doesnt guarantee that you can get BK into HFP to get through the hole